lib/components/user/styled.ts

```diff
@@ -48,6 +48,22 @@ export const TripPanelHeading = styled(Panel.Heading)`
   background-color: white !important;
 `
 
+export const TripPanelAlert = styled(Button)`
```

Comment:
For accessibility purposes this shouldn't be a button, [it should be a link! ](https://www.digitala11y.com/links-vs-buttons-a-perennial-problem/#:~:text=Use%20links%20when%20the%20user%2Daction%20doesn%E2%80%99t%20affect%20the%20website%20at%20all.%20In%20this%2C%20the%20users%20are%20just%20readers%20or%20spectators%20of%20the%20site.%20For%20example%2C%20to%20navigate%20to%20the%20next%20page%20or%20an%20external%20source%20after%20viewing%20the%20content%20of%20the%20page.)

Reply:
good catch!